testing/py-traceback2/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+pkgname=py-traceback2
```

Comment:
`python2` versions all of these 4 depends are needed so `py-bandit` can be added with a `check()` 

(which will finally allow me to fix another `python` package with a `check()` using `py-bandit`)